### DISCUSSION TITLE
为何更新到2.4.0以后就用不了了？

### DISCUSSION THREAD

**Discussion by evianincrush** [the1812/Bilibili-Evolved#3693] [提问 / Q&A] [answered]
我是不能使用新版本了吗？只能用2.2.4了

**Comment by the1812** [comment, marked as answer]
换源了, 不翻墙可以装 jsDelivr 版

![image](https://user-images.githubusercontent.com/26504152/191053962-3639f14c-5390-483c-b7b6-696856498be0.png)
